src/main/java/games/strategy/triplea/delegate/BaseEditDelegate.java

```diff
@@ -11,8 +11,8 @@
 import games.strategy.triplea.Constants;
 
 public abstract class BaseEditDelegate extends BasePersistentDelegate {
-  public static String EDITMODE_ON = "Turning on Edit Mode";
-  public static String EDITMODE_OFF = "Turning off Edit Mode";
+  private static final String EDITMODE_ON = "Turning on Edit Mode";
+  private static final String EDITMODE_OFF = "Turning off Edit Mode";
```

Comment:
These fields were not used outside this class, so I reduced their visibility appropriately as I was already modifying these lines.